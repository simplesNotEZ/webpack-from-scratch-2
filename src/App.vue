<template>
  <div>
    <h1>Hola Mundo! from webpack-from-scratch-2</h1>
    <p>Can you see me yet?</p>
  </div>
</template>
