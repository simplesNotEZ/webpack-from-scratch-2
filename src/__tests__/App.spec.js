import { shallowMount } from '@vue/test-utils';
import App from '@/App.vue';

describe('App', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(App);
    expect(wrapper).toMatchSnapshot();
  });
});
