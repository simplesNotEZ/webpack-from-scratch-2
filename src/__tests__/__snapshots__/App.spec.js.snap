// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div class="app-container">
  <perelandra-page-stub></perelandra-page-stub>
  <p class="test-p">
    You can see me ya rogerian bastard?
  </p> <span style="color: yellow; background-color: red;">And how bout now?</span>
</div>
`;
